docs/advanced/addons/vmimport.md

```diff
@@ -184,4 +185,15 @@ spec:
 
 :::note 
 OpenStack allows users to have multiple instances with the same name. In such a scenario, users are advised to use the Instance ID. The reconciliation logic tries to perform a name-to-ID lookup when a name is used.
-:::
\ No newline at end of file
+:::
+
+#### Known issues
+* **Source virtual machine name is not RFC1123 compliant**: When creating a virtual machine object, the vm-import-controller add-on uses the name of the source virtual machine, which may not meet the Kubernetes object [naming criteria](https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names). You may need to rename the source virtual machine to allow successful completion of the import.
+
+
+* **Virtual machine image name too long**: The vm-import-controller add-on labels each imported disk using the format `vm-import-$VMname-$DiskName`. If a label exceeds 63 characters, you will see the following error message in the vm-import-controller logs:
```

Comment:
This does not happen anymore in v1.5.x thanks to https://github.com/harvester/vm-import-controller/pull/63.